frontend/src/components/AddMember/index.js

```diff
@@ -190,7 +192,12 @@ export const ProfileForm = ({
         />
       )}
 
-      <Button type="warning" inputType="submit" disabled={!canGoNext()}>
+      <Button
+        onClick={() => void 0}
```

Comment:
Alternatively, we could call `submitForm` and discard the `onSubmit={submitForm}` from the form. Could we submit the form otherwise ?

Reply:
We will have the `onClick` as an optional prop in `fe-components v1.0.17`. Would that do?

Reply:
Will surely do! ;)